pkg/util/util.go

```diff
@@ -387,3 +388,30 @@ func SetVolumeOwnership(path, gid, policy string) error {
 	}
 	return volume.SetVolumeOwnership(&VolumeMounter{path: path}, path, &gidInt64, &fsGroupChangePolicy, nil)
 }
+
+// ExecFunc returns a exec function's output and error
+type ExecFunc func() (err error)
+
+// TimeoutFunc returns output and error if an ExecFunc timeout
+type TimeoutFunc func() (err error)
+
+// WaitForExecCompletion waits for the exec function to complete or times out
```

Comment:
WaitForExecCompletion waits for the exec function to complete or return timeout error

Reply:
done